src/utils/app_config.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+def database_url():
+    database_url = os.getenv("TAGGING-SUGGESTER-DATABASE-URL")
```

Comment:
Is this meant to be different to `DATABASE_URI` specified in `config.py`?

Reply:
Removed config.py

Reply:
👍